components/docs-chef-io/content/automate/security_best_practices.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Security Best Practices"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Security Best Practices"
+    parent = "automate/reference"
+    identifier = "automate/reference/security_best_practices.md Security Best Practices"
+    weight = 50
++++
+
+## Insuring autocomplete remains disable on login screen.
+
+Their is a configration maintained in automate this determine autocomplete functionlity
```

Comment:
Replace it with: A configuration maintained in automate determines to autocomplete functionality in the nginx config.